src/components/planet.js

```diff
@@ -1,61 +1,74 @@
-import React from "react"
-import PropTypes from "prop-types"
 import _random from "lodash/random"
 import _round from "lodash/round"
+import _uniqueId from "lodash/uniqueId"
+import React from "react"
+import PropTypes from "prop-types"
 
 import HoveringAnimationStyle from "./planet/hovering_animation_style"
 import RadialGradient from "./planet/radial_gradient"
-import ViewableMonitor from "./planet/viewable_monitor"
 
 import styles from "./planet.module.css"
 
 const Planet = ({
   codeName,
-  color,
+  morph,
+  hide,
   hovering,
   hoveringMin,
   hoveringMax,
-  morph,
+  color,
 }) => {
   const hoveringAmplitude = _random(hoveringMin, hoveringMax)
   const hoveringAnimationDuration = _round(_random(0.8, 1.8), 2)
-  const animationName = `planet-${codeName}-hovering`
-  const radialID = `planet-${codeName}-radial`
+  const animationName = `planet-${codeName}-hovering-${_uniqueId()}`
+  const radialID = `planet-${codeName}-radial-${_uniqueId()}`
```

Comment:
Why the unique IDs? Do we have repeating codenames?

Reply:
Was trying to fix the Safari bug.

Reply:
Also, I'm keen on using that to make sure we have no conflicts. It's a pattern I used before when dealing with `id` tags. What do you think?

The problem on safari was because the ID was repeated when the `morph` cloned the elements. As you can notice I moved that out of the `{...moprh}` element.

I think this can prevent future bugs.

Reply:
> Was trying to fix the Safari bug.

Did you succeed?

> Also, I'm keen on using that to make sure we have no conflicts. It's a pattern I used before when dealing with id tags. What do you think?
> 
> The problem on safari was because the ID was repeated when the morph cloned the elements. As you can notice I moved that out of the {...moprh} element.
> 
> I think this can prevent future bugs.

I agree with the pattern, I've used it before and I second it, but in this specific case if you look through the codebase, the `codename` is unique to every instance of the planet precisely to avoid clashings with the animations and gradients. There is no need for the unique ID here because of that.

I'm not exactly aware anymore why, but I believe there was a reason for the `codename` to be set from above, it had to be used elsewhere. I fear this unique ID might be breaking something but I no longer remember exactly what.

However if we manage to remove the `codename` in the future, I would very much prefer to change it to use this approach.

Reply:
I'll remove the `uniqueId` and I'll get back to you, to check if it's still working on Safari.